Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionLoggerManager.cs

```diff
@@ -87,11 +87,11 @@ public ExecutionLoggerConfiguration Configuration
                 if (value != null)
```

Comment:
we do it only if Value is null, do we have handelling in case user modify the ExecutionLogger path after doing first execution?